AQ3DC/README.md

```diff
@@ -0,0 +1,20 @@
+# AQ3DC : Automatic Quantification for 3D Cephalometrics
+
+Authors: Baptiste Baquero, Maxime Gillot, Lucia Cevidanes, Jean-Christophe Fillion-Robin, David Allemang
+	
+## What is it?
+
+This Slicer extension aims is to automatically computes head measurements used in craniofacial surgery. Using placed fiducials, it allows users to compute 2D angles: Yaw, Pitch and Roll; and decompose the 3D distance (between points, points to line) into the three different components: R-L , A-P and S-I. It is possible to compute the middle point between two fiducials and export the values. It is possible to compute lists of measurements selected by users for a single case or a whole study sample, at one or more time points.
+ 
+## File Formats
+
+AQ3DC allows users to compute all the measurements on points from .json files. Save middle points created in a .json file. Measurement list and results of all the computed measurements saved as .xlsx files.
+
+## Prerequisites for users
+
+The AQ3DC extension really need consistency in the data. For each patient, in the same time point and at different time point, if the landmarks are not named exactly the same the measurement for this patient won't be perform, and will just be skiped.
```

Comment:
Is this sentence related to the input json file ?

Reply:
The JSON files are standard `.mrk.json` landmark files. This is referring to the fact that AQ3DC uses simple string comparison for measurements; the landmark label must be an exact match to the point for measurement, otherwise the measurement will be skipped.